CMakeLists.txt

```diff
@@ -9,30 +9,49 @@ project(quicr-transport
 if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
     option(QTRANSPORT_BUILD_TESTS "Build tests for quicr transport" ON)
 else()
-    option(QTRANSPORT_BUILD_TESTS "Build tests for quicr trasnsport" OFF)
+    option(QTRANSPORT_BUILD_TESTS "Build tests for quicr transport" OFF)
 endif()
 
+option(QUIC_TRANSPORT "Enable QUIC Transport (Default ON)" ON)
```

Comment:
Considering selection of UDP vs QUIC vs H3 vs maybe TCP in the future is at execution time, disabling QUIC or any other transport protocol option would invalidate the execution time config and use-cases to support that.  

What was the reason to not compile in the other protocols? Is it that the dependencies with QUIC are too much causing pain for testing? If so, maybe we wrap this into TEST knobs.

Reply:
this is a temporary fix to get  esp builds going. PicoTLS has been updated to support mbed tls then we don't need this anymore.